cloud/migration-stork.md

```diff
@@ -115,14 +115,59 @@ status:
   schedulerStatus: ""
   storageStatus: ""
 ```
-5. Once you apply the above spec you should be able to check the status of the pairing. On a successful pairing, you should 
+5. (EKS Only) When pairing with an EKS cluster, you also need to pass in your
+   AWS credentials which will be used to generate the IAM token. This can be
+   achieved in 2 ways
+   1. Create a secret and mount in the stork deployment (Secure)
+       1. Create a secret in kube-system namespace with your aws credentials file:
+       ```
+       $ kubectl create secret  generic --from-file=$HOME/.aws/credentials -n  kube-system aws-creds
+       secret/aws-creds created
+       ```
+       2. Mount the secret created above in the stork deployment. Run `kubectl edit deployment -n kube-system stork` and make the following updates.
```

Comment:
maybe add the command `kubectl get po -n kube-system -l name=stork` after this to tell users to make sure stork was updated properly

Reply:
Added